paywall/src/paywall-builder/constants.js

```diff
@@ -15,3 +15,6 @@ export const POST_MESSAGE_UPDATE_ACCOUNT = 'update/account'
 export const POST_MESSAGE_UPDATE_ACCOUNT_BALANCE = 'update/accountBalance'
 export const POST_MESSAGE_UPDATE_NETWORK = 'update/network'
 export const POST_MESSAGE_UPDATE_WALLET = 'update/walletmodal'
+export const POST_MESSAGE_DATA_REQUEST = 'data/request'
```

Comment:
Could we use a better name here? Something more descriptive? I also see that for all of these other we have `verb/object`. Could we use the same syntax? (at least that would be `request/data`... but not great!)

Reply:
yes, since we're asking the chain to send any updates, I'll rename it to `POST_MESSAGE_SEND_UPDATES`